src/modules/patternReplace/patternReplace.module.ts

```diff
@@ -32,10 +52,34 @@ export const patternReplace: BotModule = {
       const hasModification = message.content !== modifiedContent;
       if (!hasModification) return;
 
-      const newMessage = [`<@${message.author.id}>`, modifiedContent].join('\n');
+      const newMessageContent = [`<@${message.author.id}>`, modifiedContent].join('\n');
+
+      const newMessage = await message.channel.send(newMessageContent);
+
+      await newMessage.react('❌');
+      await message.react('❌');
+
+      const awaitReactionsOptions: AwaitReactionsOptions = {
+        filter: (reaction, user) => reactionCollectorFilter(reaction, user, message.author.id),
+        max: 1,
+        time: 60000,
```

Comment:
add this in a descriptive constant please (const XX_MINUTE = )

Reply:
of course